includes/executing.h

```diff
@@ -1,6 +1,13 @@
 #ifndef EXECUTING_H
 # define EXECUTING_H
 
+typedef enum e_process
+{
+	MAIN = 1,
+	PARENT_PROCESS,
```

Comment:
what's the difference between the main and the parent?

Reply:
Let's call it Main.

Reply:
I just removed that enum. I'll put it back if I ever use it.